src/Makefile.am

```diff
@@ -3112,6 +3122,8 @@ tests_testEventLoop_SOURCES = \
 	carp.h \
 	tests/stub_carp.cc \
 	cbdata.cc \
+	ClientActiveRequests.h \
+	ClientActiveRequests.cc \
```

Comment:
Does testEventLoop and other similar cases actually use the ClientActiveRequests structure? If not, can it be provided via a stub file rather than explicitly adding the ClientActiveRequests concept inside unrelated test cases?

Reply:
Maybe; I'll check it.

Reply:
Stub done. I also extended stat.cc stub and used it in all tests

Reply:
I hope that stub file is not actually needed, but I am resolving _this_ change request because its primary problem has been addressed. I will deal with the new stub file in a later review.